device/globals/game.gd

```diff
@@ -357,29 +357,26 @@ func set_inventory_enabled(p_enabled):
 
 func set_camera_limits():
 	var cam_limit = camera_limits
-	if camera_limits.size.x == 0 && camera_limits.size.y == 0:
-		var p = get_parent()
+	if camera_limits.size.x == 0 and camera_limits.size.y == 0:
 		var area = Rect2()
-		for i in range(0, p.get_child_count()):
-			var c = p.get_child(i)
-			if c is preload("res://globals/background.gd"):
-				var pos = c.get_global_position()
-				var size = c.get_size()
+		for child in get_parent().get_children():
+			if child is preload("res://globals/background.gd"):
```

Comment:
This will cause #132 to need a rebase.

Reply:
If #132 is merged first, I can do the rebase.